distribution/solr-distro/pom.xml

```diff
@@ -80,12 +80,10 @@
                         </goals>
                         <configuration>
                             <url>
-                              ${solr.download.url}/lucene/solr/${solr.version}/solr-${solr.version}.zip
+                              http://artifacts.phx.connexta.com/repository/archive.apache.org/lucene/solr/8.6.1/solr-8.6.1.zip
```

Comment:
just testing to see if I can fix the Jenkins cache issue by downloading directly from nexus